trackersite/tracker/views.py

```diff
@@ -257,43 +336,74 @@ def edit_ticket(request, pk):
     if not ticket.can_edit(request.user):
         return HttpResponseForbidden(_('You cannot edit this ticket.'))
     TicketEditForm = get_edit_ticket_form_class(ticket)
-    
+
     MediaInfoFormSet = mediainfoformset_factory(extra=1, can_delete=True)
     ExpeditureFormSet = expeditureformset_factory(extra=1, can_delete=True)
-    
+    PreexpeditureFormSet = preexpeditureformset_factory(extra=1, can_delete=True)
+
     if request.method == 'POST':
         ticketform = TicketEditForm(request.POST, instance=ticket)
         try:
             mediainfo = MediaInfoFormSet(request.POST, prefix='mediainfo', instance=ticket)
-            expeditures = ExpeditureFormSet(request.POST, prefix='expediture', instance=ticket)
+            if 'content' not in ticket.ack_set():
+                expeditures = ExpeditureFormSet(request.POST, prefix='expediture', instance=ticket)
+            else:
+                expeditures = None
```

Comment:
Jsme v pohodě s tím že když někdo tiketu odškrtne content ack, tak bude mít v databázi výdaje které půjdou do všech součtů, ale nebudou vidět ve formulářích?

Reply:
Chtěl jsem zajistit, aby to nešlo editovat. Pokud existuje něco jednoduššího na bázi parametru disabled, proč to nepoužít.

Reply:
Tohle je spíš otázka na návrh, určitě to jde takhle schovat, ale trochu se bojím toho bordelu v součtech.